src/vellum/workflows/nodes/core/inline_subworkflow_node/node.py

```diff
@@ -66,4 +69,9 @@ def run(self) -> Iterator[BaseOutput]:
 
     def _compile_subworkflow_inputs(self) -> WorkflowInputsType:
         inputs_class = self.subworkflow.get_inputs_class()
-        return inputs_class(**self.subworkflow_inputs)
+        if isinstance(self.subworkflow_inputs, dict):
```

Comment:
Are we sure we want to support two forms here? If so, we will need to pick one that we want codegen to use. My 1/5 preference is to have one representation in general and my 3/5 preference is to codegen the class representation even if we support the dict repr.

Reply:
My 2/5 preference with user-facing interfaces in general is as flexible as reasonably possible on the input, as restrictive as reasonably possible in the output. In this case, `subworkflow_inputs` pointing to a JSON output of an upstream node seems like a reasonable use case to support

Agree with codegenning Inputs classes